src/Microsoft.Health.Fhir.Core.UnitTests/Features/Search/SearchValues/ReferenceSearchValueParserTests.cs

```diff
@@ -22,6 +23,8 @@ public class ReferenceSearchValueParserTests
 
         public ReferenceSearchValueParserTests()
         {
+            ModelExtensions.SetModelFactory();
```

Comment:
Let's chat about this a little bit later. The ambient context might make thigns a little bit challenging when we have STU3 and R4 tests running in parallel (potentially).

Reply:
Agree